core-devel/gcc/02-compiler/beyond

```diff
@@ -44,7 +44,7 @@ if [[ "${CROSS:-$ARCH}" != "powerpc" && "${CROSS:-$ARCH}" != "ppc64" && \
       "${CROSS:-$ARCH}" != "i486" && "${CROSS:-$ARCH}" != "armv4" && "${CROSS:-$ARCH}" != "armv6hf" && \
       "${CROSS:-$ARCH}" != "armv7hf" && "${CROSS:-$ARCH}" != "loongson2f" && \
       "${CROSS:-$ARCH}" != "mips64r6el" && "${CROSS:-$ARCH}" != "loongarch64" && \
-      "${CROSS:-$ARCH}" != "loongson3" ]]; then
+      "${CROSS:-$ARCH}" != "loongson3" && "${CROSS:-$ARCH}" != "loongarch64_nosimd" ]]; then
     abinfo "Preparing go and gofmt for alternatives ..."
```

Comment:
Use `ab_match_arch`

Reply:
I will handle this later, as this pull request is only for architecture bring up. This will take a Core update.